README.md

```diff
@@ -268,7 +268,7 @@ Or to run them manually:
 | Eigen         | 3.3.7      | MPL2                                                       | [eigen.tuxfamily.org](http://eigen.tuxfamily.org/index.php)                                                                                |
 | IAU SOFA      | 2018-01-30 | [SOFA Software License](http://www.iausofa.org/tandc.html) | [www.iausofa.org](http://www.iausofa.org)                                                                                                  |
 | SPICE Toolkit | N0066      | [NAIF](https://naif.jpl.nasa.gov/naif/rules.html)          | [naif.jpl.nasa.gov/naif/toolkit.html](https://naif.jpl.nasa.gov/naif/toolkit.html)                                                         |
-| GeographicLib | 1.49       | MIT                                                        | [geographiclib.sourceforge.io](https://geographiclib.sourceforge.io)                                                                       |
+| GeographicLib | 1.52       | MIT                                                        | [geographiclib.sourceforge.io](https://geographiclib.sourceforge.io)                                                                       |
```

Comment:
@apaletta3 Did you get a chance to dig into the following versions, what they had/fix potentially and the diff resulting in the errors we had seen when trying to install them? This is simply to ensure that:
- The following versions do not contain patches/fixes on top of `1.52` for instance. 
- We understand the changes made to that dependency which makes it incompatible with our current installation process today.